lib/robots/dor_repo/gis_assembly/normalize_data.rb

```diff
@@ -5,49 +5,98 @@ module DorRepo
     module GisAssembly
       class NormalizeData < Base
         def initialize
-          super('gisAssemblyWF', 'normalize-data', check_queued_status: true) # init LyberCore::Robot
+          super('gisAssemblyWF', 'normalize-data')
+        end
+
+        def perform_work
+          logger.debug "normalize-data working on #{bare_druid}"
+
+          rootdir = GisRobotSuite.locate_druid_path bare_druid, type: :stage
+          datafn = "#{rootdir}/content/data_EPSG_4326.zip"
+          if File.size?(datafn)
+            logger.info "normalize-data: #{bare_druid} found existing normalized data: #{File.basename(datafn)}"
+            return
+          end
+
+          File.umask(002)
+          flags = {
+            overwrite_prj: true,
+            tmpdir: Settings.geohydra.tmpdir
+          }
+
+          fn = "#{rootdir}/content/data.zip" # original content
```

Comment:
`fn` is a crappy variable name.  How about file_name or `data_zip_file_path` or something useful?

Reply:
Fixed.